scripts/kind-e2e/README.md

```diff
@@ -0,0 +1,135 @@
+```
+
+List the contexts:
+
+```bash
+kubectl config list-contexts
+```
+
+You should be able to see 3 contexts. From this stage you can interact with the clusters
+as with any normal k8s cluster.
+
+**NOTE**: Each time **make e2e status=keep** command is executed, the local code will be build, pushed to kind clusters
+as docker images, submariner will be redeployed on the clusters from pushed images and E2E tests will be executed.
+This mode allows the developers to test their local code fast on a very close to real world scenario setup.
+
+#### Logging
+
+Providing **logging=true** parameter to **make e2e** command will setup ELK stack on the kind clusters.
+The logs from three clusters will be shipped to an elasticsearch deployment on cluster1.
+Logging should be used only in addition to **status=keep** command. The default value for **logging** is **false**.
+
+```bash
+make e2e status=keep logging=true
+```
+
+To access Kibana, run the following from new terminal tab/window:
+
+```bash
+export KUBECONFIG=$GOPATH/src/github.com/rancher/submariner/output/kind-config/local-dev/kind-config-cluster1:$GOPATH/src/github.com/rancher/submariner/output/kind-config/local-dev/kind-config-cluster2:$GOPATH/src/github.com/rancher/submariner/output/kind-config/local-dev/kind-config-cluster3
+kubectl config use-context cluster1
+kibana_pod=$(kubectl get pods -l app=kibana | awk 'FNR > 1 {print $1}')
+kubectl port-forward ${kibana_pod} 8080:5601
+```
+
+Open new browser tab/window and access Kibana at http://localhost:8080.
+
+Create default index **filebeat-*** and choose **@timestamp** as time filter field. After the default index pattern is configured,
+the following lucene query example can be used to query the logs.
+
+```bash
+kubernetes.namespace:"submariner" AND kubernetes.node.name: (cluster2* OR cluster3*) AND kubernetes.labels.app: submariner*
+```
+
+#### Cleanup
+At any time you can run a cleanup command that will remove kind resources.
+
+```bash
+make e2e status=cleanup
+```
+
+You can do full docker cleanup, but it will force all the docker images to be removed and invalidate the local docker cache. 
+The next run will be a cold run and will take more time.
+
+```bash
+docker system prune --all
+``` 
+
+#### Full example
+
+```bash
+make e2e status=keep version=1.14.1 logging=true
+```
+
+<!--links-->
+[kind]: https://github.com/kubernetes-sigs/kind
+[docker]: https://docs.docker.com/install/
+[kubectl]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
+[k9s]: https://github.com/derailed/k9s
+[kubetail]: https://github.com/johanhaleby/kubetail
+[kind release page]: https://github.com/kubernetes-sigs/kind/releases
+[Change SELinux to permissive]: https://www.ibm.com/support/knowledgecenter/en/POWER8/p8ef9/p8ef9_selinux_setup.htm
+[Add local user to docker group]: https://docs.docker.com/install/linux/linux-postinstall/
+
```

Comment:
Oh!, documentation! :-)  we can probably refine it later or put it together to the main docs/devel docs, awesome! :)

Reply:
The docs definitely need refinement :) before we put it on the main doc page.